pkgs/build-support/nix-gitignore/default.nix

```
@@ -148,10 +148,10 @@ in rec {
       '');
 
   withGitignoreFile = patterns: root:
-    lib.toList patterns ++ [(root + "/.gitignore")];
+    lib.toList patterns ++ [ ".git" ] ++ [(root + "/.gitignore")];
```

Shouldn't this be

```suggestion
    lib.toList patterns ++ [(root + "/.git")] ++ [(root + "/.gitignore")];
```

?

---

I undid this after testing:
```
error: reading from file: Is a directory
```
because ".git" is a gitignore patter, not a path to a file to be read.